test/StringProcessing/Parse/regex_parse_end_of_buffer.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: %target-typecheck-verify-swift -enable-experimental-string-processing
-// REQUIRES: libswift
+// REQUIRES: swift_in_compiler
 
 // Note there is purposefully no trailing newline here.
 // expected-error@+1 {{unterminated regex literal}}
-var unterminated = '/xy
```

Comment:
This test is meant not to have a trailing newline, could you restore it?

Reply:
I think Xcode did this. Fixed.